pkg/vetter/conflictingvirtualservicehost/vet_test.go

```diff
@@ -451,7 +464,7 @@ var _ = Describe("Conflicting Virtual Service Host Vet Notes", func() {
 		})
 
 		// This test can be deleted/return a conflict if we want to report
-		// on conflicts within the same VS.
+		// on conflicts within the same VS. This is not a conflict because second one is more specific
 		It("Does not warn if two routes conflict but are in the same VS", func() {
```

Comment:
If you change the comment, you should change this to:
"Does not warn if two routes in the same VS and do not conflict"
Or something to that effect.

Reply:
will do